Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -2198,23 +2217,13 @@ namespace Babylon
                 return m_jsXRViewerPose.Value();
             }
 
-            Napi::Value GetPose(const Napi::CallbackInfo& info)
+            Napi::Value GetPoseData(const Napi::CallbackInfo& info)
             {
-                if (info[0].IsExternal())
-                {
-                    const auto& space = *info[0].As<Napi::External<xr::Space>>().Data();
-                    m_transform.Update(space, false);
-                    return m_jsPose.Value();
-                }
-                else
-                {
-                    auto* xrSpace = XRReferenceSpace::Unwrap(info[0].As<Napi::Object>());
-                    assert(xrSpace != nullptr);
-                    Napi::Object napiPose = XRPose::New(info);
-                    XRPose* pose = XRPose::Unwrap(napiPose);
-                    pose->Update(xrSpace->GetTransform());
-                    return std::move(napiPose);
-                }
+                const auto& space = *info[0].As<Napi::External<xr::Space>>().Data();
+                auto vectorBuffer = info[2].As<Napi::ArrayBuffer>();
```

Comment:
What about `info[1]`?

Reply:
We currently only support one reference space, which is stored in info[1] and is assumed to be the identity with respect to the provided space

Reply:
Maybe assert or throw exception when it's not?